Src/Base/AMReX_ParallelDescriptor.H

```diff
@@ -1167,6 +1167,17 @@ Gather(const T& t, int /*root*/)
     return resl;
 }
 
+template <class T>
+void
+Gatherv (const T* send, int sc,
+         T* recv, const std::vector<int>& /*rc*/,
+         const std::vector<int>& /*disp*/, int /*root*/)
+{
+    std::vector<T> resl(sc);
+    for (int j=0; j<sc; ++j) { recv[j] = send[j]; }
+    return resl;
```

Comment:
This function's return type is `void`, not `std::vector`.

Reply:
Oops. Too many small PRs. Went too fast. :)
Fixed.